src/components/ActionBar/ActionBar.js

```diff
@@ -194,7 +192,7 @@ const ActionBar = ({
       const newFilters = { ...filters, ...formValues }
 
       if (filtersHelperResult) {
-        if (params.name || params.funcName || params.hash) {
+        if (params.name || params.hash) {
```

Comment:
check if `params.hash` is still needed. It was only for functions but functions doesn't have `hash` in the URL anymore

Reply:
also check other usage of the `params.hash`